build.sh

```diff
@@ -91,7 +91,7 @@ usage() {
     echo "  -V VERSION      - Override version (default: run gen-version)"
     echo "  -R RELEASE      - Override release tag (default: '1pdns', do not include %{dist} here)"
     echo "  -m MODULES      - Build only specific components (comma separated; warning: this disables install tests)"
-    echo "  -e EPOCH        - Set a specific Epoch for packages"
+    echo "  -e EPOCH        - Set a specific Epoch for RPM packages"
```

Comment:
This is also used for the epoch in Debian packages. Epoch in that context is the number before the `:` in `1:3.4-1pdns`

Reply:
OK, I will change it. I just wanted to make clear it has nothing to do with `SOURCE_DATE_EPOCH`.

How about something like this?

```suggestion
    echo "  -e EPOCH        - Set a specific Epoch for RPM/DEB packages, to reset the package versioning"
```

Reply:
perhaps mention the word 'version'?

Reply:
"set a specific version epoch"

Reply:
Like this?

```suggestion
    echo "  -e EPOCH        - Set a specific version Epoch for RPM/DEB packages"
```

Reply:
yes, do it!